plugins/cpp_metrics/test/sources/service/mccabe.cpp

```diff
@@ -0,0 +1,148 @@
+void conditionals(int arg1, bool arg2) { // +1
```

Comment:
I'm not sure why this file and many other code related to PR #689 appear here. @mcserep do you think this will mess up the commit history?

Reply:
@dbukki mentioned that he rebased the branch. I think something went the wrong way, since code already accepted and merged into `master` shows up in this PR as new changes. This also makes the review not easy.

@dbukki please clean this up and either force push the branch with a clean version history (containing only the changes related to your PR), or create a new PR accordingly.

Reply:
After merging the current `master` into this branch, unrelated changes to this PR was removed from the diff correctly. This issue is resolved.